### DISCUSSION TITLE
Plz help me with module selection

### DISCUSSION THREAD

**Discussion by elicohen12** [community/community#65991] [Projects and Issues]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello,

I have to make betting website.
It doesn't have complicated components
But site must look good.
which moudle is best, do you think?
Plz help me

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by cohen1280** [comment]
I think, MUI is best
Of curse, MUI will not need because your website doesn't have complicated component.
But betting website must look good.
So I suggeset MUI

**Reply by elicohen12** [reply]
Thank you.

**Comment by quyetlv12** [comment]
You can use tailwind css

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
